pages/doc/logging_log_browser.md

```diff
@@ -21,7 +21,73 @@ To use the Log Browser:
 
 ![An annotated screenshot of the Log Browser.](images/logging_log_browser_annotated_screen.png)
 
-{%include note.html content="Tanzu Observability retains logs for 7, 15, or 30 days, based on your settings."%}
+{%include note.html content="Tanzu Observability retains logs for 7, 15, or 30 days, based on your settings. For longer retention needs, contact [technical support](wavefront_support_feedback.html#support)."%}
```

Comment:
Consider removing contact technical support because it's not possible to retain logs for longer than 30 days.

Reply:
I will be keeping it as discussed :) There is JIRA to track this.